tests/platform_tests/test_link_down.py

```diff
@@ -22,7 +22,7 @@
     pytest.mark.topology('t2'),
 ]
 
-MAX_TIME_TO_REBOOT = 120
+MAX_TIME_TO_REBOOT = 200
```

Comment:
@wenyiz2021 can we have a check so that it does not dilute non chassis timeout?

Reply:
@prgeor thanks for bringing up this! I initially set it larger because this test is running on chassis only by now
I'm working on another way, so **chassis** duts could overwrite timeout values with what was defined in their inventory file.